synapse/lib/stormlib/model.py

```diff
@@ -692,3 +693,203 @@ async def _lock(self, name, locked):
         todo = s_common.todo('setDeprLock', name, locked)
         gatekeys = ((self.runt.user.iden, ('model', 'deprecated', 'lock'), None),)
         await self.runt.dyncall('cortex', todo, gatekeys=gatekeys)
+
+@s_stormtypes.registry.registerLib
+class LibModelMigration(s_stormtypes.Lib):
+    '''
+    A Storm library containing migration tools.
+    '''
+    _storm_locals = (
+        {'name': 'copyData', 'desc': 'Copy node data from one node to the write layer for another node.',
+         'type': {'type': 'function', '_funcname': '_methCopyData',
+                  'args': (
+                      {'name': 'src', 'type': 'node', 'desc': 'The node to copy data from.', },
+                      {'name': 'dst', 'type': 'node', 'desc': 'The node to copy data to.', },
+                      {'name': 'overwrite', 'type': 'boolean', 'default': False,
+                       'desc': 'Copy data even if the key exists on the destination node.', },
+                  ),
+                  'returns': {'type': 'null', }}},
+        {'name': 'copyEdges', 'desc': 'Copy edges from one node to the write layer for another node.',
```

Comment:
Since the function assumes the top layer of the current view ( like most ) i think including `to the write layer` here might be more confusing than helpful. Perhaps `Copy edges from the src node to the dst node.` here and below?

Reply:
I can update that.  I didn't know if it would be confusing since generally our migrations are either run on the layers or in view deporder, while these won't be.  Do you think it needs a "in the current view" specifier?

Reply:
Updated docstrings per discussion.